internal/controllers/clusterclass/clusterclass_controller_test.go

```diff
@@ -545,3 +549,49 @@ func TestReconciler_reconcileVariables(t *testing.T) {
 		})
 	}
 }
+
+func TestReconciler_extensionConfigToClusterClass(t *testing.T) {
+	extConfig := &runtimev1.ExtensionConfig{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "runtime1",
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ExtensionConfig",
+			APIVersion: runtimev1.GroupVersion.String(),
+		},
+		Spec: runtimev1.ExtensionConfigSpec{
+			NamespaceSelector: &metav1.LabelSelector{},
+		},
+	}
+
+	expected := []reconcile.Request{}
+	// These ClusterClasses will be reconciled as they both reference the passed runtimeExtension `discover-variables` call.
```

Comment:
```suggestion
	// These ClusterClasses will be reconciled as they both reference the passed ExtensionConfig `runtime1`.
```
I think. Sorry I don't really get the current doc.
(similar in l.578)